src/components/ListHeader/Title/Title.scss

```diff
@@ -0,0 +1,5 @@
+.Title {
+    @media screen and (max-width: $breakpoint-mobile) {
+      text-align: center;
+    }
+}
```

Comment:
No capitalizing classname, this file also needs a new line

<img width="176" alt="Screen Shot 2021-01-26 at 9 38 18 AM" src="https://user-images.githubusercontent.com/13306885/105882189-39033780-5fba-11eb-8d6e-88802cec2da3.png">